__init__.py

```diff
@@ -360,18 +360,40 @@ def selectLanguage( self, path ):
 		elif conf["viur.languageMethod"] == "url":
 			tmppath = urlparse.urlparse( path ).path
 			tmppath = [ urlparse.unquote( x ) for x in tmppath.lower().strip("/").split("/") ]
+			langSet = False
 			if len( tmppath )>0 and tmppath[0] in conf["viur.availableLanguages"]+list( conf["viur.languageAliasMap"].keys() ):
 				self.language = tmppath[0]
-				return( path[ len( tmppath[0])+1: ] ) #Return the path stripped by its language segment
+				langSet = True
```

Comment:
This line is obsole: You're setting langSet to True and return from the function afterwards.